parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -303,7 +303,7 @@ installSpinKube(){
 
     for version in "${package_versions[@]}"; do
         containerd_spinkube_url=$(evalPackageDownloadURL ${PACKAGE_DOWNLOAD_URL})
-        downloadSpinKube $download_location $containerd_spinkube_url "v$version" # adding v to version for simplicity
+        downloadSpinKube $download_location $containerd_spinkube_url "$version"
```

Comment:
Where does the $version value come from - I didn't notice us add the "v" back in anywhere, so this might be a regression.

Reply:
It's from components.json and it actually has a 'v' already in that url. 
"downloadURL": "https://acs-mirror.azureedge.net/spinkube/v${version}/linux/${CPU_ARCH}"

Reply:
I will keep the 'v' for now, though I found there is a bug for that. But I wanted to keep this PR pure for shellcheck.